client/apps/edit/components/admin/test/index.coffee

```diff
@@ -22,6 +22,7 @@ describe 'EditAdmin', ->
         EditAdmin::setupAuthorAutocomplete = sinon.stub()
         EditAdmin::setupFairAutocomplete = sinon.stub()
         EditAdmin::setupPartnerAutocomplete = sinon.stub()
+        EditAdmin::setupAuctionAutocomplete = sinon.stub()
```

Comment:
Nah in this case we'd just be testing a bunch of configuration options—not a lot of code to test there.